Source/OCMockTests/OCMockObjectMacroTests.m

```diff
@@ -450,4 +450,11 @@ - (void)testCanUseMacroToStubMethodWithAnyNonObjectArgumentChainedWithOCMStubRec
     XCTAssertEqualObjects(@"f", [mock commonPrefixWithString:@"foo" options:NSCaseInsensitiveSearch]);
 }
 
+- (void)testShouldThrowAssertionWhenStoppedMockHasClassMethodStubbed
+{
+	id mock1 = OCMClassMock([TestClassWithClassMethods class]);
```

Comment:
As opposed to `mock2`? Maybe just `mock` here?

Reply:
Fixed. Thanks for catching that.